modules/social_features/social_private_message/src/Service/SocialPrivateMessageService.php

```diff
@@ -63,41 +63,46 @@ public function deleteUserDataThreadInfo(EntityBase $entity) {
    *   The number of unread threads.
    */
   public function updateUnreadCount() {
-    $unread = 0;
+    $unread = &drupal_static(__FUNCTION__);
 
-    // Get the user.
-    $uid = $this->currentUser->id();
-    /** @var \Drupal\user\UserStorageInterface $user_storage */
-    $user_storage = $this->userManager;
-    /** @var \Drupal\user\UserInterface $user */
-    $user = $user_storage->load($uid);
+    if (!isset($unread)) {
```

Comment:
This is the only actual change together with line `66`, the rest is indentation.